components/NavigationBar/examples/NavigationBarWithFlatButtonItemExample.m

```diff
@@ -0,0 +1,91 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
It was just code copied from another example.

1. Since I do not plan to submit this code, I guess it's not a big issue.
1. Existing code might have old year in copyright section. Is this an issue?